### DISCUSSION TITLE
Question regarding adding replacement preset

### DISCUSSION THREAD

**Discussion by karfau** [renovatebot/renovate#15585] [Request Help] [answered]
I just found out about [replacement presets](https://docs.renovatebot.com/presets-replacements/) and am wondering what the criteria are for adding one.

My specific case that I would like to propose is a replacement of the npm package `xmldom` to it's scoped version `@xmldom/xmldom` because of https://github.com/xmldom/xmldom/issues/271

Would a PR that adds such a replacement be accepted?

**Comment by rarkins** [comment, marked as answer]
Yes, please go ahead: https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/replacements.ts

**Reply by karfau** [reply]
It's ready. #15588 

I have some more questions regarding replacements in general, ist there some "general documentation" or blog post about them?
Or should I file a new discussion for those?
